github/ci/prow/files/jobs/kubevirt/kubevirt/kubevirt-periodics.yaml

```diff
@@ -118,12 +121,25 @@ periodics:
           git clone https://github.com/kubevirt/kubevirt.git &&
           cd kubevirt &&
           export DOCKER_PREFIX='kubevirtnightlybuilds' &&
-          export DOCKER_TAG='latest' &&
+          export DOCKER_TAG="$(date +%Y%m%d)_$(git show --format=%h)" &&
           make &&
-          make push
+          make push &&
+          git show --format=%H > _out/commit &&
+          bucket_dir="kubevirt-prow/devel/nightly/release/$(date +%Y%m%d)" &&
+          gsutil cp ./_out/manifests/release/kubevirt-operator.yaml gs://$bucket_dir/kubevirt-operator.yaml &&
+          gsutil cp ./_out/manifests/release/kubevirt-cr.yaml gs://$bucket_dir/kubevirt-cr.yaml &&
+          gsutil cp ./_out/commit gs://$bucket_dir/commit
```

Comment:
Additionally inject a `latest` file that points to `kubevirt-prow/devel/nightly/release/kubevirt/kubevirt/$(date +%Y%m%d)`